.gitignore

```diff
@@ -58,3 +58,9 @@ yarn-debug.log
 # Ignore Docker option files
 docker-compose.override.yml
 
+# Ignore plugin files
+/plugins/*
+!/plugins/.keep
```

Comment:
you don't need to have these in gitignore, just add them with `git add -f` (and now that they're in the index, it doesn't matter anyway)

Reply:
Cool, will do